File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/ApplicationFeatureModel.ts
import { ApplicationModel } from "./applicationModel";
import { ApplicationFeaturePermissionModel } from "./applicationFeaturePermissionModel";
import { AppRoleFeatureModel } from "./appRoleFeatureModel";

export interface ApplicationFeatureModel {
    featueId: number;
    applicationId: number | null;
    featueCode: string;
    featueNameEn: string;
    featueNameAr: string;
    featuePath: string;
    icon: string;
    color: string;
    displyOrder: number | null;
    isActive: boolean | null;
    isNew: boolean | null;
    isUpdated: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;

}

File: SofiForce.Models/Models/EntityModels/ApplicationFeatureModel.cs
using System;

namespace Models
{
    public class ApplicationFeatureModel
    {
        public int FeatueId { get; set; }
        public int? ApplicationId { get; set; }
        public string FeatueCode { get; set; }
        public string FeatueNameEn { get; set; }
        public string FeatueNameAr { get; set; }
        public string FeatuePath { get; set; }
        public string Icon { get; set; }
        public string Color { get; set; }
        public int? DisplyOrder { get; set; }
        public bool? IsActive { get; set; }
        public bool? IsNew { get; set; }
        public bool? IsUpdated { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}
